src/sentry/api/endpoints/group_details.py

```diff
@@ -179,39 +179,18 @@ def get(self, request, group):
         :auth: required
         """
         # TODO(dcramer): handle unauthenticated/public response
-
-        # TODO(jess): This can be removed when tagstore v2 is deprecated
-        use_snuba = request.GET.get('enable_snuba') == '1'
```

Comment:
are we passing this field now?  Same as for group_tags. Before ignoring the query argument we should verify in nginx that everyone is passing 1.